components/settingPreference/SettingPreference.tsx

```diff
@@ -0,0 +1,151 @@
+import { Dropdown } from 'react-native-element-dropdown';
+import { phonedark } from '../icons/icons';
+
+const Settings = () => {
+  const [emailNotifications, setEmailNotifications] = useState(false);
+  const [pushNotifications, setPushNotifications] = useState(false);
+  const [selectedTheme, setSelectedTheme] = useState('system');
+
+    const colorScheme = selectedTheme === 'system' ? useColorScheme() : selectedTheme;
+    const textStyle = colorScheme === 'dark' ? 'text-gray-100' : 'text-secondary-light-900';
+    const borderColor = colorScheme === 'dark' ? 'border-gray-700' : 'border-gray-300';
+    const containerStyle = colorScheme === 'dark' ? 'bg-primary-dark' : 'bg-primary-light';
+
+  
+  const themeData=[
+    {label: "System", value: "system"},
+    {label: "Light", value: "light"},
+    {label: "Dark", value: "dark"}
+  ]
+  return (
+    <View className={`p-4 mb-8 ${containerStyle}`}>
+      <Text className={`text-2xl font-extrabold ml-4 mb-4 ${textStyle}`}>Settings</Text>
+
+      {/* Profile Section */}
+      <View className="mb-6 p-4 rounded-lg flex flex-row justify-center">
+        <View className="flex-1 mr-4">
+          <Text className={`text-xl font-bold ${textStyle}`}>Profile</Text>
+          <Text className={`text-lg mt-2 font-light ${textStyle}`}>
+            Edit profile, export account, data…
+          </Text>
+        </View>
+        <TouchableOpacity className="flex items-end">
+          <Text
+            className={`${textStyle} mt-2`}
+            onPress={() => router.push('/dashboard/trainee/Profile')}
+          >
+            Change
+          </Text>
+        </TouchableOpacity>
+      </View>
+
+      {/* Theme Picker */}
+      <View className={`mb-6 p-4 border-t ${borderColor} rounded-lg flex flex-row`}>
+        <View className="flex-1 mr-4">
+          <Text className={`text-xl font-bold ${textStyle}`}>Appearance</Text>
+          <Text className={`text-lg mt-2 ${textStyle}`}>Theme preferences</Text>
+        </View>
+        <View className="flex-row items-center gap-5">
+          <Dropdown
+            labelField="label"
+            valueField="value"
+            data={themeData}
+            value={selectedTheme}
+            onChange={(item) => {
+            }}
+            placeholder="Select Theme"
+            selectedTextStyle={{ color: colorScheme=== 'dark'? '#fff' : "#000" }}
+            //@ts-ignore
+            style={styles.picker(colorScheme)}
+          />
+        </View>
+      </View>
+
+      {/* Language Picker */}
+      <View className={`mb-6 p-4 border-t ${borderColor} rounded-lg flex justify-between`}>
+        <View className="flex-1 mr-4">
+          <Text className={`text-xl font-bold ${textStyle}`}>Language</Text>
+          <Text className={`${textStyle} mt-2 text-lg`}>Language Preferences</Text>
+        </View>
+        <View className="mt-4">
+          <LanguagePicker showFlag={false} />
+        </View>
+      </View>
```

Comment:
try this code and see if it look the same as the one we have on figma

{/* Language Picker */}
      <View className={`mb-6 p-4 border-t ${borderColor} rounded-lg  justify-between`}>
      <Text className={`text-xl font-bold ${textStyle}`}>Language</Text>
        <View className="flex-row justify-between  w-full mr-4">
          
          <Text className={`${textStyle} mt-2 text-lg flex-1`}>Language Preferences</Text>
          <View className={`border  ${borderColor} w-44 flex-1 rounded-lg mt-4`}>
          <LanguagePicker showFlag={false} />
        </View>
        </View>
        
      </View>
      <img width="180" src="https://github.com/user-attachments/assets/dc74331a-9c29-401c-8951-be61193d1018" />